auth-tokens/src/main/java/com/palantir/tokens/auth/BearerToken.java

```diff
@@ -49,4 +58,8 @@ public static BearerToken valueOf(String token) {
     public final String toString() {
         return getToken();
     }
+
+    public final JsonWebToken asJwt() {
```

Comment:
so is every bearertoken a JWT? if so, then why not bake this assumption into the BearerToken directly?